tgui/packages/tgui/interfaces/Pacman.js

```diff
@@ -0,0 +1,174 @@
+import { useBackend } from "../backend";
+import { Button, Box, Section, ProgressBar, LabeledList, Knob, NumberInput, Flex, Grid } from "../components";
+import { Window } from "../layouts";
+
+export const Pacman = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    broken,
+    anchored,
+    active,
+    fuel_type,
+    fuel_usage,
+    fuel_stored, // current fuel level
+    fuel_cap, // fuel_capacity
+    is_ai,
+    tmp_current, // temperature_current
+    tmp_max, // temperature_max
+    tmp_overheat, // temperature_overheat
+    output_max,
+    power_gen,
+    output_set,
+  } = data;
+  let fuelRatio = fuel_stored / fuel_cap;
+  let tmpRatio = tmp_current / tmp_max; // temperature ratio
+  let output_watts = output_set * power_gen;
```

Comment:
```suggestion
  let output_watts = (output_set * power_gen).toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');;
```
Either use something like this to add a thousands separator, or just show the value in kW.
Otherwise you get output like "200000 W".

Reply:
Take a look at `tgui\packages\tgui\interfaces\Smes.js` and make usre of `formatPower()` please

Reply:
done